config/am-scripts/scripts-config.json

```diff
@@ -708,6 +708,54 @@
                 "lastModifiedDate": 1617813675725
             },
             "filename": "ch-update-legacy-password-check-original.js"
+        },
+        {
+            "payload": {
+                "_id": "188d3f9d-ca04-4df7-bcd7-eed3fe27e21e",
+                "name": "CH - Login Error Message",
+                "description": "Create error message structure for login page",
+                "script": "<replace>",
+                "default": false,
+                "language": "JAVASCRIPT",
+                "context": "AUTHENTICATION_TREE_DECISION_NODE",
+                "createdBy": "null",
+                "creationDate": 0,
+                "lastModifiedBy": "null",
+                "lastModifiedDate": 0
+            },
+            "filename": "ch-login-error-message.js"
+        },
+        {
+            "payload": {
+                "_id": "064705f3-93d3-482c-ad3f-4531a28ce0a0",
+                "name": "CH - Callback Login Error",
+                "description": "Handle error message structure for login page",
+                "script": "<replace>",
+                "default": false,
+                "language": "JAVASCRIPT",
+                "context": "AUTHENTICATION_TREE_DECISION_NODE",
+                "createdBy": "null",
+                "creationDate": 0,
+                "lastModifiedBy": "null",
+                "lastModifiedDate": 0
+            },
+            "filename": "ch-callback-login-error.js"
```

Comment:
I try to use a naming convention where the first part of the file is the journey the scripts belong to (e.g. ch-login-error.js)
If the script is used in more journeys, forget about this comment, leave it generic

Reply:
I think it'll end up being generic - will leave for now.